### PR TITLE
SISRP-13261, Welcome to CalCentral for delegate users

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.berkeley.edu/browse/SISRP-13261
https://jira.berkeley.edu/browse/SISRP-13381

* New card for Toolbox page. See PDF attached to https://jira.berkeley.edu/browse/SISRP-6601
* Hide delegate cards when feature flag

### CODE REVIEW

File: src/assets/javascripts/angular/controllers/widgets/delegateIntroController.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  $scope.delegateIntro = {
+    isLoading: true,
+    showComplete: false
+  };
+
+  var loadInformation = function() {
+    $scope.delegateIntro.isLoading = false;
+  };
+
+  loadInformation();
```
Comment (paulkerschen) [inline src/assets/javascripts/angular/controllers/widgets/delegateIntroController.js]: @johncrossman If I'm reading right, when this controller is loaded, `$scope.delegateIntro.isLoading` will be initialized as true, then set to false an eyeblink later. Is there any need for that pattern here? Generally we use it to set the spinner directive while APIs are loading from the back end, but it seems we have no such APIs for this card.

File: src/assets/templates/widgets/toolbox/delegate_intro_to_calcentral.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
+<div
+  class="cc-widget cc-toolbox-widget"
+  data-ng-controller="DelegateIntroController"
+  data-cc-spinner-directive="delegateIntro.isLoading"
```
Comment (paulkerschen) [inline src/assets/templates/widgets/toolbox/delegate_intro_to_calcentral.html]: @johncrossman See above; might be we don't need this.

File: src/assets/javascripts/angular/controllers/widgets/delegateIntroController.js (hunk 1 above)
Comment (johncrossman) [inline src/assets/javascripts/angular/controllers/widgets/delegateIntroController.js]: Yes, you're right @paulkerschen I'm going to close this PR in favor of simpler setup. Thank you.

Comment (johncrossman) [inline src/assets/javascripts/angular/controllers/widgets/delegateIntroController.js]: New PR is #4774

**Pull request closed without merging**